src/renderers/shared/fiber/ReactFiberRoot.js

```diff
@@ -25,6 +25,8 @@ export type FiberRoot = {
   isScheduled: boolean,
   // The work schedule is a linked list.
   nextScheduledRoot: FiberRoot | null,
+  // Track nested commits, to prevent an infinite loop.
+  nestedUpdateCount: number,
```

Comment:
Needs to be added to createFiberRoot below to prevent deopting.

(Maybe we should add a preventExtensions here like we have in createFiber https://github.com/facebook/react/blob/5ee72dcd821e1706a26a24d27feb9bbb9bde3b90/src/renderers/shared/fiber/ReactFiber.js#L205.)

Reply:
Flow also caught this error, I just forgot to run it